socorro/lib/util.py

```diff
@@ -19,7 +19,7 @@ def dotdict_to_dict(sdotdict):
 
     def _dictify(thing):
         if isinstance(thing, collections.Mapping):
-            return dict([(key, _dictify(val)) for key, val in thing.items()])
+            return {key: _dictify(val) for key, val in thing.items()}
```

Comment:
I did this while I was looking at this code. It's not really related to the rest of the PR.